pelican/plugins/obsidian/obsidian.py

```diff
@@ -125,20 +144,33 @@ def populate_files_and_articles(article_generator):
         full_path, filename_w_ext = os.path.split(article)
         filename, ext = os.path.splitext(filename_w_ext)
         path = str(full_path).replace(str(base_path), '') + '/'
+        # For windows
+        if os.sep == '\\':
+            path = path.replace('\\', '/')
 
         # This work on both pages and posts/articles
         ARTICLE_PATHS[filename] = path
 
+    __log__.debug('Found %d articles', len(ARTICLE_PATHS))
+
     # Get list of all other relavant files
```

Comment:
Not your typo, but would be nice to fix :)

Reply:
done.